### DISCUSSION TITLE
Failed to load resource: the server responded with a status of 403 ()

### DISCUSSION THREAD

**Discussion by RaonHaonKIM** [kimytsc/covid-rest-vaccine-macro#31] [Q&A] [answered]
콘솔에 모두 입력 후에 돌리면 저런 문구가 콘솔에서 뜨는데 정상적으로 가동되고 있는 건가요??

**Comment by kimytsc** [comment]
혹시 네이버, 카카오 중에 어떤걸로 하셨나요?

**Reply by RaonHaonKIM** [reply]
카카오 사용했습니다!

**Reply by kimytsc** [reply]
혹시 DevTools의 Network에서 어떤 URL이 403 에러를 띄우고 있는지 확인 가능할까요?

**Reply by RaonHaonKIM** [reply]
![1](https://user-images.githubusercontent.com/83588964/128795615-63e8a3b2-35c5-416c-bf7b-210b1812890f.JPG)
![2](https://user-images.githubusercontent.com/83588964/128795617-3fb9938e-b4cb-4c87-b180-ba55fa29a4d3.JPG)
이렇습니다!

**Reply by thfekaql** [reply]
카카오 사용중입니다 저도 같은 오류 문구가 콘솔창에 나옵니다..

**Comment by kimytsc** [comment, marked as answer]
/api/v2/reservation 403 로그는 예약 시도 실패시 나타나는 로그입니다.

카카오는 예약 시도 실패시 403 상태코드로 응답을 주는데, 이 과정에서 무조건적으로 `Failed to load resource: the server responded with a status of 403 ()`이 로그에 노출되게 됩니다.

화면에서 병원명과 실패사유가 변경된다면, 정상적입니다.

**Reply by thfekaql** [reply]
콘솔창에 POST https://vaccine.kakao.com/api/v2/reservation 403 이렇게 뜨는 것도 정상인가요?

**Reply by kimytsc** [reply]
콘솔창을 켜놓은 상태에서 시도된 경우에는 `POST https://vaccine.kakao.com/api/v2/reservation 403`으로 노출이 됩니다.

콘솔창을 껐다가 다시 켜보시면 `Failed to load resource: the server responded with a status of 403 ()` 메세지로 바뀌어 노출되어 있는걸 보실 수 있을꺼에요.

**Reply by thfekaql** [reply]
걱정 덜었습니다 감사합니다
